bn-models/src/models/_common/observer-assessment.ts

```diff
@@ -0,0 +1,22 @@
+import { Base } from "../_base";
+export const ObserverAssessmentTypeName = 'observer-assessment';
+import { CouchID, BoatnetDate } from ".";
+import {AssessmentQuestion, AssessmentAnswerSet } from "../_lookups";
+
+export interface AssessmentResponse {
+    question?: AssessmentQuestion;
+    response?: string;
+}
+
+export interface ObserverAssessment extends Base {
+    evaluationPeriod: CouchID;
+    vesselCount?: number;
+    tripCount?: number;
```

Comment:
Just curious what are these for? Are they used to display info on the UI?

Reply:
They probably don't really need to be stored, as the display values are calculated based on state info every time the assessment page loads, just figured since I had them I might as well store them in the assessment doc.